src/main/resources/templates/notifications/actions-and-notifications.html

```diff
@@ -31,28 +31,28 @@ <h1 class="govuk-heading-l">
-                <th scope="col" class="govuk-table__header">LAA Ref</th>
-                <th scope="col" class="govuk-table__header">Provider ref</th>
-                <th scope="col" class="govuk-table__header">Client</th>
-                <th scope="col" class="govuk-table__header">Fee earner</th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'assignDate','Date assigned')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'subject','Subject')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'dueDate','Due date')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'userName','Assigned to')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'status','Status')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'caseReferenceNumber','LAA Ref')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'providerCaseReferenceNumber','Provider ref')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'clientName','Client')}"></th>
+                <th th:replace="~{notifications/partials/sorting :: sorting(${notifications},'feeEarner','Fee earner')}"></th>
             </tr>
             </thead>
             <tbody class="govuk-table__body">
             <tr class="govuk-table__row" th:each="notification : ${notifications.content}">
-                <td class="govuk-table__cell" th:text="${#dates.format(notification.notificationDetail.assignDate,'dd-MM-yyyy')}"></td>
-                <td class="govuk-table__cell" th:text="${notification.notificationDetail.subject}"></td>
-                <td class="govuk-table__cell" th:text="${#dates.format(notification.notificationDetail.dueDate,'dd-MM-yyy')}"></td>
-                <td class="govuk-table__cell" th:text="${notification.user.userLoginId}"></td>
-                <td class="govuk-table__cell" th:text="${notification.notificationDetail.status}"></td>
-                <td class="govuk-table__cell" th:text="${notification.caseReferenceNumber}"></td>
-                <td class="govuk-table__cell" th:text="${notification.providerCaseReferenceNumber}"></td>
-                <td class="govuk-table__cell" th:text="${notification.clientName}"></td>
-                <td class="govuk-table__cell" th:text="${notification.feeEarner}"></td>
+                <td class="govuk-table__cell govuk-!-font-size-17" th:text="${#dates.format(notification.assignDate,'dd-MM-yyyy')}"></td>
+                <td class="govuk-table__cell govuk-!-font-size-17" th:text="${notification.subject}"></td>
+                <td class="govuk-table__cell govuk-!-font-size-17" th:text="${#dates.format(notification.dueDate,'dd-MM-yyy')}"></td>
```

Comment:
Should it be slashed not hyphens?
The rest of PUI deisplays as dd/mm/yyyy

Reply:
Done - and corrected the year format to `yyyy`